tests/manage/z_cluster/test_ceph_default_values_check.py

```diff
@@ -25,6 +25,7 @@
 
 @tier1
 @skipif_external_mode
+@skipif_ocs_version("<4.9")
```

Comment:
Add it to individual test (test_noobaa_postgres_cm_post_ocs_upgrade), else it will skip for all tests under this class

Reply:
Done